spanner/api/Program.cs

```diff
@@ -187,11 +189,14 @@ class ReadWriteWithTransactionOptions
         public string instanceId { get; set; }
         [Value(2, HelpText = "The ID of the database where the sample database resides.", Required = true)]
         public string databaseId { get; set; }
+        [Value(3, HelpText = "The platform code to execute (netcore or net45).", Required = false)]
+        public string platform { get; set; }
     }
 
     public class Program
     {
         static readonly ILog s_logger = LogManager.GetLogger(typeof(Program));
+        private static readonly string NetCorePlatform = "netcore";

```

Comment:
I forget, will const work here?

It will have exactly the same effect, but be a stronger hint to the user that it's a constant.